CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 All notable changes to this project will be documented in this file.
 
+## [1.11.1] - 2024-10-24
```

Comment:
Do I also need to update `version.txt` file or will that happen in CI?

Reply:
yes that file should be updated.

v1.11.1 was already [released](https://github.com/passageidentity/passage-go/releases), but it looks like the changelog was forgotten

the overall changelog should probably be something like

```
## [1.11.2] - 2024-10-24

### Added

- chore: LICENSE file added

### Changed

- docs: README updated
- docs: update Passage Docs link
- ci: pin gorename to v0.24.0 to avoid conflict with go runtime 1.20.0
- test: fix test user identifier conflict

## [1.11.1] - 2024-07-29

### Changed

- chore(deps): bump github.com/lestrrat-go/jwx from 1.2.26 to 1.2.29 
```

Reply:
Updated - lmk if this looks good